scripts/Tools/preview_run

```diff
@@ -56,6 +56,7 @@ def _main_func(description):
         print("  total tasks: {}".format(case.total_tasks))
         print("  tasks per node: {}".format(case.tasks_per_node))
         print("  thread count: {}".format(case.thread_count))
+        print("  ngpus per node: {}".format(case.ngpus_per_node))
```

Comment:
Thanks @amametjanov for this comment. This print statement actually does not work properly at this moment (i.e., it will always print `ngpus per node: 0` even if the `NGPUS_PER_NODE` is set to > 0) and I am not sure what exactly causes this issue.